src/components/frontend-engine/types.ts

```diff
@@ -86,6 +87,7 @@ export type TFrontendEngineFieldSchema<V = undefined> =
 	| IResetButtonSchema
 	| ISelectSchema<V>
 	| ISubmitButtonSchema
+	| ISwitchSchema
```

Comment:
sorry missed this, need to pass generics:

```suggestion
	| ISwitchSchema<V>
```

Reply:
done